etc/eagle_config.json

```diff
@@ -4,18 +4,9 @@
         "HOST": "0.0.0.0",
         "DEBUG": true
     },
-
-    "global": {
-        "title": "Fuel DevOps Portal",
-        "logo_txt": "Fuel DevOps",
-        "name": "Fuel DevOps",
-        "logo": "logo.png",
-        "css": "main.css"
-    },
-
     "cloud_status": {
         "enabled":  true,
-        "enabled_dashboards": ["availability", "health"],
+        "endpoint": "http://dummy.example.org/api/cloud_status",
```

Comment:
There is no cloud status service, there are 3 services one for availability, another for performance, third for health. 

But for UI guys it is simpler to have one cloud status API.